http2_test.js

```diff
@@ -0,0 +1,14 @@
+var axios = require('./index.js');
+var fs = require('fs');
+
+axios.defaults.experimental_http2 = true;
```

Comment:
this line doesn't do anything right now, it's sort of the idea, to have some kind of config boolean that can enable http2